MaterialLibrary/src/main/res/values/attrs_elevation_delegate.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="ElevationDelegate">
+        <attr name="elevation" />
+        <attr name="cornerRadius" format="dimension" />
+        <attr name="showShadows">
```

Comment:
I'd go with `shownShadows`.

`shadowVisibility` was the first idea, because it resembles `visibility`, but on second thought that's bad (they're unrelated; former is a flag, second enumone is an enum...). As a plus, `ElevationDelegate` already has the corresponding method named `setShownShadows`. :)